pydatastructs/graphs/algorithms.py

```diff
@@ -531,15 +553,15 @@ def strongly_connected_components(graph, algorithm):
     Examples
     ========
 
-    >>> from pydatastructs import Graph, AdjacencyListGraphNode
-    >>> from pydatastructs import strongly_connected_components
-    >>> v1, v2, v3 = [AdjacencyListGraphNode(i) for i in range(3)]
-    >>> g = Graph(v1, v2, v3)
-    >>> g.add_edge(v1.name, v2.name)
-    >>> g.add_edge(v2.name, v3.name)
-    >>> g.add_edge(v3.name, v1.name)
-    >>> scc = strongly_connected_components(g, 'kosaraju')
-    >>> scc == [{'2', '0', '1'}]
+    from pydatastructs import Graph, AdjacencyListGraphNode
+    from pydatastructs import strongly_connected_components
+    v1, v2, v3 = [AdjacencyListGraphNode(i) for i in range(3)]
+    g = Graph(v1, v2, v3)
+    g.add_edge(v1.name, v2.name)
+    g.add_edge(v2.name, v3.name)
+    g.add_edge(v3.name, v1.name)
+    scc = strongly_connected_components(g, 'kosaraju')
+    scc == [{'2', '0', '1'}]
```

Comment:
Please retain `>>>`. These are required for doctests to run.

Reply:
done